src/pages/Admin/AdminVenuePreview.tsx

```diff
@@ -145,14 +145,16 @@ export const AdminVenuePreview: React.FC<AdminVenuePreviewProps> = ({
               />
             </div>
           </div>
-          <div style={{ width: "150px" }}>
-            <div className="title" style={{ width: "150px" }}>
-              {PLAYA_VENUE_NAME} icon
+          {venue.mapIconImageUrl && (
+            <div style={{ width: "150px" }}>
+              <div className="title" style={{ width: "150px" }}>
+                {PLAYA_VENUE_NAME} icon
+              </div>
+              <div className="content">
+                <img className="icon" src={venue.mapIconImageUrl} alt="icon" />
+              </div>
             </div>
-            <div className="content">
-              <img className="icon" src={venue.mapIconImageUrl} alt="icon" />
-            </div>
-          </div>
+          )}
```

Comment:
We don't want to just render this icon when it's present, we want to remove it from the code entirely. So this block of code should probably be deleted.

Reply:
> As part of removing this, we should also be removing the concept of it from our core types/codebase.
> 
> If I remember correctly, I believe @denisdimitrov may have already done some work on this/potentially had a WIP branch for it?
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/internal-sparkle-issues/issues/710#issuecomment-839195504_

---

It also appears to be used in the following locations:

- https://github.com/sparkletown/sparkle/search?q=mapIconImageUrl

For each of these locations, we will need to assess how it is being used in the code there, and remove any redundant/no longer required code. It might be worth syncing with @mike-lvov about this to double check/confirm what you intend to remove once you look through the locations and formulate your plan.

---

Of particular note, removing it from our `BaseVenue` type is likely going to highlight all of the other places in the app where it exists but shouldn't:

- https://github.com/sparkletown/sparkle/blob/staging/src/types/venues.ts#L124

Reply:
> It also appears to be used in the following locations:
> 
> https://github.com/sparkletown/sparkle/search?q=mapIconImageUrl
> For each of these locations, we will need to assess how it is being used in the code there, and remove any redundant/no longer required code. It might be worth syncing with @mike-lvov about this to double check/confirm what you intend to remove once you look through the locations and formulate your plan.
> 
> Of particular note, removing it from our BaseVenue type is likely going to highlight all of the other places in the app where it exists but shouldn't:
> 
> https://github.com/sparkletown/sparkle/blob/staging/src/types/venues.ts#L124

@oleksii-front I don't think you addressed all of the above?

Reply:
@oleksii-front Once again, please follow our standards and post the commit hash + a message in review comment threads communicating when/where they are fixed, and mark it as resolved. You have been asked to do this by both @mike-lvov and myself multiple times now, and we shouldn't have to keep reminding you/asking you to do this. It make things much harder and slower for the reviewer when you don't do this.

Please don't make us need to ask you to do this again in future.